### PR TITLE
Add comment about using "template" architectures,  minor copy changes

### PR DESCRIPTION
**Pull request by diegopetrucci**

### CODE REVIEW

File: demo.md [hunk 1]
```diff
@@ -24,9 +24,10 @@ Before proceeding to the section of your choice, please consider the following t
...
+* You can use any 3rd party libraries you wish (`Alamofire`, `ReactiveCocoa`, `PromiseKit`, `Realm`, etc) but be prepared to justify why you did so. Feel free to use package managers to handle them.
 * **Be consistent in your code**. We advise using something like [raywenderlich's swift style guide](https://github.com/raywenderlich/swift-style-guide) while doing the demo. It's absolutly fine to use any other style, as long as you are consistent.
-* Clean the file project structure (if you are creating one from scratch) and remove any unused methods (e.g., from `AppDelegate`). This shows attention to detail.
+* Clean the file project structure and remove any unused methods (e.g., from `AppDelegate`). This shows attention to detail.
+* If you have to use VIPER please give it your own spin. Even though it's a good architecture often times it carries a lot of boilerplate which makes all tests that use it look the same.
```
Comment (RuiAAPeres) [inline demo.md]: I don't agree with this. What does "give it your own spin" means?

Comment (diegopetrucci) [reply demo.md]: I wanted to include a comment @sergdort made, that many VIPER projects don't have a `soul`. But I don't know how to say that without using a religious term.

Comment (sergdort) [reply demo.md]: What I meant  is that it seems like many people using VIPER most likely use module generator tools, which doesn’t show how they reason about a problem

File: demo.md [hunk 2]
```diff
@@ -62,24 +63,24 @@ The following requirements should be met:
...
-We would be happy if you would submit a project you already have (for instance, a demo project for another company). Still, the project **must** obey the following requirements:
+We would be happy if you would submit a project you already have (for instance, a demo project for another company). Still, the project **must**:
 
 * Use Swift 3.0 or above.
-* It should have at least two distinct network calls.
-* It should parse the network response and present the information to the user.
-* It should have some sort of persistence mechanism.
-* It should compile and run.
-* It should have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).
+* Have two distinct network calls.
```
Comment (wltrup) [inline demo.md]: at least two

File: demo.md [hunk 3]
```diff
@@ -62,24 +63,24 @@ The following requirements should be met:
...
+* Parse the network response and present the information to the user.
+* Have some sort of persistence mechanism.
+* Compile and run.
+* Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).
 
-If you have a project with these requirements, then perfect! Please ensure that you also have a description of what the project does, in order to give us some context.
+If you have a project with these requirements, then perfect! Please ensure that you also have a description of what the project does in order to give us some context.
```
Comment (wltrup) [inline demo.md]: I disagree with removing that comma. "in order to give us some context" is a secondary sentence that is not directly tied to the preceding sentence and, therefore, should be preceded by a comma.

File: demo.md [hunk 4]
```diff
@@ -62,24 +63,24 @@ The following requirements should be met:
...
 
 Once again, please note that the requirements above are **mandatory**.
 
 ### 3. Open Source work
 
-We would like to see a **non-trivial** pull request you have made to a public open source project. This should be something you are proud of and where you show your technical skills. **It should also be related to iOS development** and aligned with what you will do on a day-to-day basis. 😊✨🌳
+We would like to see a **non-trivial** pull request you have made to a public open source project. This should be something you are proud of that shows your technical skills. **It should also be related to iOS development** and aligned with what you will do on a day-to-day basis. 😊✨🌳
```
Comment (wltrup) [inline demo.md]: The suggested change sounds weird to my ears when read aloud. I prefer the original.

File: demo.md (hunk 1 above)
Comment (wltrup) [inline demo.md]: "If you have to use VIPER please give it your own spin. Even though it's a good architecture often times it carries a lot of boilerplate which makes all tests that use it look the same."

I don't think we should have this sentence.

**Review by wltrup — CHANGES_REQUESTED** [review]

File: demo.md (hunk 1 above)
Comment (diegopetrucci) [reply demo.md]: @dmcrodrigues is suggesting to use
```Be opinionated regarding any architecture you use, take your time to make it a little yours and a reflex of your logical thought.```
cc @wltrup

File: demo.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ Before proceeding to the section of your choice, please consider the following t
 * You can use any 3rd party libraries you wish (`Alamofire`, `ReactiveCocoa`, `PromiseKit`, `Realm`, etc) but be prepared to justify why you did so. Feel free to use package managers to handle them.
 * **Be consistent in your code**. We advise using something like [raywenderlich's swift style guide](https://github.com/raywenderlich/swift-style-guide) while doing the demo. It's absolutly fine to use any other style, as long as you are consistent.
 * Clean the file project structure and remove any unused methods (e.g., from `AppDelegate`). This shows attention to detail.
-* If you have to use VIPER please give it your own spin. Even though it's a good architecture often times it carries a lot of boilerplate which makes all tests that use it look the same.
+* Be opinionated regarding any architecture you use, take your time to make it a little yours and a reflection of your logical thought.
```
Comment (wltrup) [inline demo.md]: "Be opinionated regarding any architecture you use, take your time to make it a little yours and a reflection of your logical thought."  -> "Be opinionated regarding any architecture you use and take your time to make it a reflection of your thought process."

File: demo.md [hunk 6]
```diff
@@ -24,9 +24,10 @@ Before proceeding to the section of your choice, please consider the following t
...
-* You can use any 3rd party libraries you wish (`Alamofire`, `ReactiveCocoa`, `PromiseKit`, `Realm`, etc). In case you do, be prepared to justify why you chose them. You can use `CocoaPods`, `Carthage` or `Git Submodules` for this (please don't drag and drop stuff around).
+* You can use any 3rd party libraries you wish (`Alamofire`, `ReactiveCocoa`, `PromiseKit`, `Realm`, etc) but be prepared to justify why you did so. Feel free to use package managers to handle them.
 * **Be consistent in your code**. We advise using something like [raywenderlich's swift style guide](https://github.com/raywenderlich/swift-style-guide) while doing the demo. It's absolutly fine to use any other style, as long as you are consistent.
-* Clean the file project structure (if you are creating one from scratch) and remove any unused methods (e.g., from `AppDelegate`). This shows attention to detail.
+* Clean the file project structure and remove any unused methods (e.g., from `AppDelegate`). This shows attention to detail.
+* Be opinionated regarding any architecture you use, take your time to make it a little yours and a reflection of your logical thought.
```
Comment (wltrup) [inline demo.md]: "Be opinionated regarding any architecture you use, take your time to make it a little yours and a reflection of your logical thought." -> "Be opinionated regarding any architecture you use and take your time to make it a reflection of your thought process."

**Review by wltrup — CHANGES_REQUESTED** [review]

File: demo.md (hunk 6 above)
Comment (diegopetrucci) [reply demo.md]: done

**Review by wltrup — APPROVED** [review]

**Pull request merged**